compose/ui/ui-uikit/src/commonMain/kotlin/androidx/compose/ui/utils/Accessibility.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.uikit.utils
+
+abstract class Accessibility {
```

Comment:
Do we need a common library and API? Can we just publish uikit target?

Reply:
common code goes for free with uikit target, so feel free to remove it. 
Not sure, but there is can be a requirement to have at least 1 blank file in the directory.